### ISSUE TITLE
Some confusion on Annotation has been submitted

### ISSUE THREAD

**Issue by ARWattam** [PATRIC3/patric3_website#614]
You submitted an annotation and you want to submit a second.  You start doing the second annotation job, but the message persists, which confuses the users.  It stays the whole time while you are working on the second job.  It confused me, too.
![screen shot 2016-03-08 at 10 33 07 am](https://cloud.githubusercontent.com/assets/11475316/13608308/d2b81858-e519-11e5-8ee0-804d88dd7f6d.png)

**Comment by hyoo** [comment]
I agree. Here is my suggestion. The notification shows up as a dialog (with close button) and once the dialog is closed, the form should be reset and ready to take next job.

**Comment by nconrad** [comment]
As part of the workspace work, I moved and restyled the notifications for operations such as creating a new folder, deleting objects, etc.  Another option here would be to use the same notifications.  I.e.: 

![ef5ab50e-3731-11e7-9027-9e76fdbee635](https://cloud.githubusercontent.com/assets/1194246/26018270/69697b90-3733-11e7-8f7e-8d652c21fff9.png)

**Comment by rkenyon** [comment]
I think that is a great option.  Neal, is this something you can add?  Assigning ticket to you.

**Comment by dawenx** [comment]
@nconrad i'm going through some old issues, what's status on this?

**Comment by nconrad** [comment]
status: not implemented.

**Comment by mshukla1** [comment]
@ARWattam, do you still consider this to be a problem?
